docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -337,8 +337,13 @@ The image to run in the container.
 It is recommended to use a fully qualified image name rather than a short name, both for
 performance and robustness reasons.
 
-The format of the name is the same as when passed to `podman run`, so it supports e.g., using
-`:tag` or using digests guarantee a specific image version.
+The format of the name is the same as when passed to `podman pull`. So, it supports using
+`:tag` or digests to guarantee the specific image version.
+
+As a special case, if the `name` of the image ends with `.image`, Quadlet will use the image
+pulled by the corresponding `.image` file, and the generated systemd service contains
+a dependency on the `$name-image.service`.
+Note that the corresponding `.image` file must exist.
```

Comment:
That is _really_ _really_ _REALLY_ cool!

Reply:
Thanks